7.7.поиск_ошибок_и_ревью_кода/7.7.8.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())
+product = 1
+while n > 0:
+    digit = n % 10
+    product = product * digit
```

Comment:
Не забывай о расширенных операторах присваивания:
```suggestion
    product *= digit
```